QRCoder/QRCoder.csproj

```diff
@@ -1,63 +1,80 @@
+		<DefineConstants Condition="'$(TargetFramework)' == 'net6.0-windows'">$(DefineConstants);NET6_0_WINDOWS</DefineConstants>
+		<CheckEolTargetFramework>false</CheckEolTargetFramework>
+		<DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+		<NoWarn />
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+		<PackageId>QRCoder</PackageId>
+		<Version>1.4.3</Version>
+		<Authors>Raffael Herrmann</Authors>
+		<PackageOwners>Raffael Herrmann</PackageOwners>
+		<AssemblyName>QRCoder</AssemblyName>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+		<PackageProjectUrl>https://github.com/codebude/QRCoder/</PackageProjectUrl>
+		<PackageIcon>nuget-icon.png</PackageIcon>
+		<PackageReadmeFile>nuget-readme.md</PackageReadmeFile>
+		<PackageTags>c# csharp qr qrcoder qrcode qr-generator qr-code-generator</PackageTags>
+		<RepositoryUrl>https://github.com/codebude/QRCoder.git</RepositoryUrl>
+		<RepositoryType>git</RepositoryType>
+		<Description>QRCoder is a simple library, written in C#.NET, which enables you to create QR codes.</Description>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<None Include="Assets\nuget-icon.png" Pack="true" PackagePath="\" />
+		<None Include="Assets\nuget-readme.md" Pack="true" PackagePath="\" />
+	</ItemGroup>
+
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net35' or '$(TargetFramework)' == 'net40' ">
+		<Reference Include="PresentationCore" />
+		<Reference Include="PresentationFramework" />
+		<Reference Include="WindowsBase" />
+	</ItemGroup>
+
+	<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' or '$(TargetFramework)' == 'netstandard2.0' ">
+		<PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0" />
+	</ItemGroup>
+
+	<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net5.0-windows' ">
+		<PackageReference Include="System.Drawing.Common" Version="5.0.3" />
+	</ItemGroup>
 	<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
 		<PackageReference Include="System.Drawing.Common" Version="6.0.0" />
+		<PackageReference Include="SkiaSharp">
+			<Version>2.88.3</Version>
+		</PackageReference>
+		<PackageReference Include="SixLabors.ImageSharp" Version="3.0.1" />
 	</ItemGroup>
+	<ItemGroup Condition="'$(TargetFramework)' == 'net5.0-windows'">
+		<PackageReference Include="SkiaSharp">
+			<Version>2.88.3</Version>
+		</PackageReference>
+		<PackageReference Include="SixLabors.ImageSharp" Version="3.0.1" />
+	</ItemGroup>
+	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+		<PackageReference Include="SkiaSharp">
+			<Version>2.88.3</Version>
+		</PackageReference>
+		<PackageReference Include="SixLabors.ImageSharp" Version="3.0.1" />
```

Comment:
ImageSharp 3.0.1 has a conditionally commercial license, can this be a problem?
https://github.com/SixLabors/ImageSharp/blob/v3.0.1/LICENSE

Reply:
This is a interesting question, I am not an expert about this but below is my thought

For my understanding, QRCoder is a open source package under MIT license, which is able to apply apache license.
For developer using QRCoder, ImageSharp is serve as Transitive Package so it may also apply apache license.

```
  Works in Source or Object form are licensed to You under the Apache License, Version 2.0 if.

   - You are consuming the Work in for use in software licensed under an Open Source or Source Available license.
   - You are consuming the Work as a Transitive Package Dependency.
   - You are consuming the Work as a Direct Package Dependency in the capacity of a For-profit company/individual with
     less than 1M USD annual gross revenue.
   - You are consuming the Work as a Direct Package Dependency in the capacity of a Non-profit organization
     or Registered Charity.

   For all other scenarios, Works in Source or Object form are licensed to You under the Six Labors Commercial License
   which may be purchased by visiting https://sixlabors.com/pricing/.
```

Reply:
The guy in medium says that if the dependency depends on something with the license that isn't compliant it is a reason to worry https://medium.com/@vovabilonenko/licenses-of-npm-dependencies-bacaa00c8c65. Probably it will be good to move the potentially dangerouse dependency to the separate package. Or to downgrade image sharp. Probably it is something that maintainers will decide